backend/src/main/java/se/chalmers/dat265/group1/WebApi.java

```diff
@@ -20,25 +22,38 @@
  */
 public class WebApi {
 
-
     private static CustomerAPI ci;
     private static DigitalPartAPI dpi;
     private static MaterialInterface mi;
-    private static PrintingInterface pi;
+    private static PrintingAPI pi;
+    private static PhysicalAPI phi;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the "phi" field and declare it as a local variable in the relevant methods. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1450)